.github/workflows/README.md

```diff
@@ -0,0 +1,10 @@
+### Jobs
+
+- `lint`: Uses `vale` to pefrom plain language linting
```

Comment:
At minimum, there should be a linting here because I typed `pefrom` instead of `perform`.